platform/security/idp/security-idp-client/src/test/java/org/codice/ddf/security/idp/client/AssertionConsumerServiceTest.java

```diff
@@ -225,7 +226,8 @@ public void testProcessBadSamlResponse() throws Exception {
     String badRequest =
         Resources.toString(Resources.getResource(getClass(), "/SAMLRequest.xml"), Charsets.UTF_8);
 
-    Response response = assertionConsumerService.processSamlResponse(badRequest, RELAY_STATE_VAL);
+    Response response =
```

Comment:
❓ This test, where we're processing a bad response, is the only one testing a redirect signed response. Is it possible to craft another test for that case?